### PR TITLE
use testng p2 feature

### PR DESCRIPTION
**Pull request by missedone**
for #287 , de-bundle TestNG jars with TestNG P2 repo

### CODE REVIEW

**Comment by cbeust** [comment]
I don't really feel qualified to comment on this code review, so I'll defer to you and whoever else wants to review.

I am interested in the effect this change will have on users, though, can you give a quick explanation of that?

**Comment by missedone** [comment]
this change is almost transparent to users.
only people use "TestNG Library" for their eclipse projects could be impacted by the bug (if any) of this PR.

**Comment by missedone** [comment]
hi @mbooth101, this might be a head up to you when working on #228.
we're de-bundle the testng jars into a P2 repo: https://github.com/testng-team/testng-p2
also, it'll be appreciated if you can have a review on the OSGified testng bundles: https://dl.bintray.com/testng-team/testng-p2/
thx

**Comment by mbooth101** [comment]
@missedone Thanks for thinking of me :-) However there doesn't seem to be any p2 repo at https://dl.bintray.com/testng-team/testng-p2/ and when I visit in my browser I just see "The requested path was not found."

**Comment by missedone** [comment]
oops, i was rebuilding the update-site, could you take a look again: https://dl.bintray.com/testng-team/testng-p2/

File: pom.xml [hunk 1]
```diff
@@ -167,6 +167,11 @@
             <url>http://download.eclipse.org/releases/luna</url>
         </repository>
         <repository>
+            <id>testng-p2-repo</id>
+            <layout>p2</layout>
+            <url>http://dl.bintray.com/testng-team/testng-p2/</url>
```
Comment (missedone) [inline pom.xml]: for now, we are using the beta update site, will switch to the release repo when testng-p2 public released.

**Pull request merged**